### DISCUSSION TITLE
Add robots.txt to web server of azuracast

### DISCUSSION THREAD

**Discussion by pawelzet** [AzuraCast/AzuraCast#5755] [Q&A]
Hello,

I'm here to ask what would be the best/easiest way to add robots.txt file to my azuracast web server. I don't want robots to index my public websites - they are available just for testing purposes. It should be available on https://mystationurl.com/robots.txt. Overwriting docker-compose? How? Im able to enter container with ssh and add robots.txt file in www/web directory - it works well but I would like to keep this configuration after reboot etc. Thank you in advance for your help.

**Comment by Moonbase59** [comment]
I’d also be interested in that. I didn’t succeed to mount this file into the web root using `docker-compose.override.yml`.

**Comment by scysys** [comment]
I also agree to that.

@BusterNeece Hopefully its ok to mark you 😂
Maybe you should also consider adding a default robots.txt file that you ship with the installation.

I would prefer something like this.

```
# Block all web crawlers and search engine spiders from AzuraCast
User-agent: *

# Disallow all pages and directories from AzuraCast
Disallow: /

# Allow resources under the /public/ directory to make sure Stations can be indexed
Allow: /public/
```

This should be okay for all station owners. It blocks the panel itself but allows access to the public pages for the stations.

**Comment by Moonbase59** [comment]
Thanks for implementing this, much appreciated!

**Reply by pawelzet** [reply]
@Moonbase59 How you achieved that? You mounted it in docker-compose-override.yml?

**Reply by BusterNeece** [reply]
The robots.txt that was suggested above is the one we use in the application now.

**Reply by Moonbase59** [reply]
Yup:

![Auswahl_283](https://github.com/AzuraCast/AzuraCast/assets/3706922/ad5463b0-2cb4-45e0-94e5-d7d4fa8e14c6)
